forge-gui/res/cardsfolder/upcoming/master_chef.txt

```diff
@@ -0,0 +1,11 @@
+Name:Master Chef
+ManaCost:2 G
+Types:Legendary Enchantment Background
+S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddReplacementEffects$ This & Other | Description$ Commander creatures you own have "This creature enters the battlefield with an additional +1/+1 counter on it" and "Other creatures you control enter with an additional +1/+1 counter on them."
+SVar:This:Event$ Moved | ValidCard$ Card.Self | Destination$ Battlefield | ReplaceWith$ ExtraCounter | Description$ This creature enters the battlefield with an additional +1/+1 counter on it.
+SVar:Other:Event$ Moved | ValidCard$ Creature.Other+YouCtrl | Destination$ Battlefield | ReplaceWith$ ExtraCounter | Description$ Other creatures you control enter with an additional +1/+1 counter on them.
+SVar:ExtraCounter:DB$ PutCounter | ETB$ True | Defined$ ReplacedCard | CounterType$ P1P1 | SubAbility$ ETB
+SVar:ETB:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Defined$ ReplacedCard
```

Comment:
@Hanmac 
shouldn't this use InternalEtbReplacement?

Reply:
yeah use InternalEtbReplacement for now

you also can still use etbCounter until i remove it